arrow-libs/ank/build.gradle

```diff
@@ -12,9 +12,9 @@ buildscript {
 
 plugins {
     id "org.jetbrains.kotlin.jvm" version "$KOTLIN_VERSION"
-    id "org.jlleitschuh.gradle.ktlint" version "$KTLINT_GRADLE_VERSION" apply false
```

Comment:
are we dropping the properties for these versions or does this module need special overrides?

Reply:
You cannot access `gradle.properties` from `gradle.kts` but you can from `.gradle`...
However, for us to be able to leverage dependabot for automatic version updating we're going to abandon the pattern of version in `gradle.properties` and we'll use regular `groupId:artifactid:version` for dependencies.

This way we dependabot will be automatically able to create PRs for all version updates, this will happen asap but over multiple PRs. We also want to fully migrate to `gradle.kts` but in small increments, and without blocking any releases.

Reply:
You should be able to access the properties in `gradle.properties` via `properties["some.property"]` (and there are more ways, but I don't remember them now).

There are a few more options:

1. refreshVersions plugin
2. Version Catalogs + migrate from dependabot to Renovate (I was using dependabot, I migrated recently to Renovate and it is not only more powerful, but the support is also just really amazing).

I haven't used refreshVersion plugin, but it has a task to automigrate everything, maybe you can try it. For the second option, I am using it in a lot of projects and I can help you with the implementation.

Reply:
Hey @JavierSegoviaCordoba,

Thanks for commenting! I'm really not so good with this stuff 😅
Version Catalogs looks great I hadn't heard of it before.

If you're interested in contributing Arrow could definitely use some improvements in Gradle. There are no tickets for it but feel free to make some, but we're trying to get maintenance down as much as possible whilst still being able to do frequent releases with important version updates. So if we can get powerful automatic version updating that would be awesome!

I'll try your suggestion, thank you!

Reply:
Yeah, I can contribute @nomisRev. If you like the Version Catalog + renovate approach, I can create a PR adding them and tell you how to activate renovate, what do you think if we talk in kotlinlang slack to organize it?

Reply:
Yes, a contribution would of course be super appreciated 🙏

Reply:
I've created [this one]( https://github.com/arrow-kt/arrow/issues/2532) :eyes: